erpnext_documentation/www/docs/user/manual/en/automation/event_streaming.md

```diff
@@ -119,11 +124,46 @@ In the Field Mapping child table:
 
 ![Document Type Mapping](/docs/assets/img/automation/event-field-mapping.png)
 
-- **Is Child Table**: If the field you are trying to map is a child table, you need to check this.
-- **Child Table Mapping**: Create another Document Type Mapping for the child table fields and select it here.
+#### 3.5.2 Default Value for some field
+
+If your field is not mapped to any other remote fieldname and you always want the field to have the same value, set the default value field. Event if you have set the remote fieldname, in case during the sync, remote field's value is not found and the "Default Value" has been specified, it will be set.
```

Comment:
remote field's value is not found and if the "Default Value" has been specified, it will be set.